### PR TITLE
Extend ArrayAssertions and enhance tests

### PR DESCRIPTION
**Pull request by chanseokoh**

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/ArrayAssertions.java [hunk 1]
```diff
@@ -25,26 +25,43 @@
...
+    assertSize(message, expecetedSize, arr, Objects::toString, 120);
+  }
+
+  public static <T> void assertSize(int expecetedSize, T[] arr, Function<T, String> printer) {
+    assertSize(null, expecetedSize, arr, printer, 120);
+  }
+
+  /**
+   * Assert that {@code arr} has the expected size; if not, fail with a message showing as many
+   * elements will fit until the total message size exceeds {@code maxLength}.
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/ArrayAssertions.java]: as will fit

File: plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/ArrayAssertions.java [hunk 2]
```diff
@@ -25,26 +25,43 @@
...
-   * Assert that {@code arr} is empty; if not, fail with a message showing as many elements will fit
-   * until the total message size exceeds {@code maxLength}.
-   */
   public static <T> void assertIsEmpty(String message, T[] arr, Function<T, String> printer,
       int maxLength) {
+    assertSize(message, 0, arr, printer, maxLength);
+  }
+
+  public static <T> void assertSize(int expecetedSize, T[] arr) {
+    assertSize(null, expecetedSize, arr, Objects::toString, 120);
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/ArrayAssertions.java]: expectedSize

File: plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/ArrayAssertions.java [hunk 3]
```diff
@@ -25,26 +25,43 @@
...
+  }
+
+  public static <T> void assertSize(String message, int expecetedSize, T[] arr) {
+    assertSize(message, expecetedSize, arr, Objects::toString, 120);
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/ArrayAssertions.java]: expectedSize

File: plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/ArrayAssertions.java [hunk 4]
```diff
@@ -25,26 +25,43 @@
...
+  }
+
+  public static <T> void assertSize(int expecetedSize, T[] arr, Function<T, String> printer) {
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/ArrayAssertions.java]: expectedSize

Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/ArrayAssertions.java]: arr --> array

**Comment by chanseokoh** [comment]
Kokoro failed due to transient Eclipse repo problem. Re-triggering with an empty commit.

**Pull request merged**